conf/deployuidb.sh

```
@@ -7,24 +7,59 @@ host="$3"
+        MYSQL='sudo greatdb'
+    fi
+fi
+
 base=`dirname $0`
 flyway="$base/tools/flyway-3.2.1/flyway"
 flyway_sql="$base/tools/flyway-3.2.1/sql/"
 
 # give grant option to the new management ip after `zstack-ctl change_ip`
-mysql --user=$user --password=$password --port=$port << EOF
-grant all privileges on *.* to root@"$host" identified by "$password" with grant option;
+if command -v greatdb &> /dev/null; then
```

Shellcheck (0.11.0)

[warning] In POSIX sh, &> is undefined.

(SC3020)